crates/paralegal-policy/src/context.rs

```diff
@@ -165,45 +273,40 @@ impl Context {
     }
 
     /// Returns whether a node or set of nodes that match some condition flows to a node or set of nodes that match another condition through the configured edge type.
-    pub fn flows_to(
-        &self,
-        ctrl_id: Option<ControllerId>,
-        src: &DataSource,
-        sink: &CallSiteOrDataSink,
-        edge_type: EdgeType,
-    ) -> bool {
-        let ctrl_flow_ids = match &ctrl_id {
-            Some(id) => vec![id],
-            None => self.flows_to.keys().collect(),
+    pub fn flows_to(&self, src: &CtrlNode, sink: &CtrlNode, edge_type: EdgeType) -> bool {
```

Comment:
`CtrlNode` is small (e.g. implements `Copy`) so we don't have to use references here. We can just pass `CtrlNode` for both `src` and `sink`.

Reply:
Done